docs/application/dotnet/guides/nui/quickpanelclient.md

```diff
@@ -0,0 +1,101 @@
+# Quickpanel Client
+
+Quickpanel is a library to control the Quickpanel service window that shows notifications and system setup widgets. You can use Quickpanel to get various information from the Quickpanel service window. For example, media player application needs to close the Quickpanel service window during playback of video. In this case, you can use the Quickpanel client.
+
+## Prerequisites
+To use the methods of the Quickpanel API, use `Tizen.NUI.WindowSystem` in your application:
```

Comment:
How about changing "use" to "include"?

To use the methods of the Quickpanel API, include the \`Tizen.NUI.WindowSystem\` in your application:

Reply:
The word is similar but `Tizen.NUI.WindowSystem` is namespace, not header.
So, I think "use" is more better then "include"